ProjectBank/Server.Integration.Tests/CustomWebApplicationFactory.cs

```diff
@@ -0,0 +1,102 @@
+namespace Server.Integration.Tests;
+
+public class CustomWebApplicationFactory : WebApplicationFactory<Program>
```

Comment:
CustomWebApplicationFactory might not be the best name

Reply:
I think this is fine since its job is to create the test application. Mr. Microsoft does the exact same thing